library-core/src/main/java/com/mikepenz/fastadapter/adapters/ModelAdapter.java

```diff
@@ -235,7 +204,7 @@ public int getAdapterItemCount() {
      */
     @Override
     public List<Item> getAdapterItems() {
```

Comment:
Best solution would be to give back just mItems, but that would require the IItemList to implement List

Reply:
I've seen this done in a few places throughout the app. Is there a specific reason for allowing the user to access the private list? [In general that's a bad idea](http://www.informit.com/articles/article.aspx?p=31551&seqNum=2).

Reply:
It would break existing code.

Reply:
A breaking logical change (vs a binary incompatible change) sounds like a good idea for a minor version bump 😁